content/articles/Developing-simple-Blockchain-app-In-flutter/index.md

```diff
@@ -0,0 +1,216 @@
+
+In this article, we will write about creating a blockchain flutter application.
+### Introduction
+ A flutter application is a  mobile app dart code that runs on iOS and Android.  Flutter uses Google's Skia library to draw widgets in UIs, which gives it high performance. Flutter also has its advanced framework that lets developers create mobile apps easily and quickly
+
+We will be using the blockchain library to create a simple application that will allow users to create and manage their blockchain.
+### 	Prerequisites 
+- Have Visual studio installed.
+- Be familiar with flutter and dart language.
+### Creating a Blockchain Flutter Application
+
+To create a blockchain flutter application, we need to first install the Flutter SDK. You can find the installation instructions on the Flutter website.
+
+Once you have installed the Flutter SDK, you can create a new project by running the following command in your terminal:
+```cmd
+flutter create my-blockchain-app
+```
+Now that our project is created, let's open it in your favorite editor.  I will be using Visual Studio Code for this tutorial.
+
+You should see the following code in your editor:
+
+//This is a dart file.  If you open it in another editor, make sure to associate *.dart files with dart-sdk
```

Comment:
I am a bit lost by this and the following statement. Wrap that error in a code-block so it is easy to understand